affirm/build.gradle

```diff
@@ -1,13 +1,15 @@
+import com.github.spotbugs.snom.SpotBugsTask
+
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-android'
 apply plugin: 'pmd'
-apply plugin: 'findbugs'
+apply plugin: 'com.github.spotbugs'
```

Comment:
Just curious: Is spotbugs better than findbugs?

Reply:
@kevindavmoore https://docs.gradle.org/current/userguide/upgrading_version_5.html#the_findbugs_plugin_has_been_removed

The deprecated FindBugs plugin has been removed. As an alternative, you can use the SpotBugs plugin from the Gradle Plugin Portal.